posts/REDCapTidieR_v0.3.qmd

```diff
@@ -0,0 +1,103 @@
+---
+title: REDCapTidieR 0.3.0
+image: images/REDCapTidieR.png
+author: Stephan Kadauke, Ezra Porter, Richard Hanna
+date: "2023-02-28"
+description: "REDCapTidieR v0.3.0 release announcement and change descriptions."
+categories:
+  - REDCap
+  - REDCapTidieR
+  - R
+---
+
+We're thrilled to announce the release of [REDCapTidieR](https://chop-cgtinformatics.github.io/REDCapTidieR/) v0.3.0 on CRAN! REDCapTidieR provides a user-friendly way to [import](https://chop-cgtinformatics.github.io/REDCapTidieR/articles/glossary.html#import) data from a [REDCap](https://www.project-redcap.org/) [project](https://chop-cgtinformatics.github.io/REDCapTidieR/articles/glossary.html#project) into the R environment.
+
+You can install the current version from CRAN with:
+
+```{r, eval = FALSE}
+install.packages("REDCapTidieR")
+```
+
+## Repeating events support 🎉
+
+### What this means in REDCap
+
+One of the biggest updates to REDCapTidieR is support for REDCap "repeating events." For the uninitiated, REDCap allows for two types of repeating elements: [**instruments**](https://chop-cgtinformatics.github.io/REDCapTidieR/articles/glossary.html#instrument) and [**events**](https://chop-cgtinformatics.github.io/REDCapTidieR/articles/glossary.html#event). Note that each of these is only available to [longitudinal](https://chop-cgtinformatics.github.io/REDCapTidieR/articles/glossary.html#longitudinal-project) [databases](https://chop-cgtinformatics.github.io/REDCapTidieR/articles/glossary.html#database). When designing a REDCap [database](https://chop-cgtinformatics.github.io/REDCapTidieR/articles/glossary.html#database), developers have the option to specify whether an event [repeats](https://chop-cgtinformatics.github.io/REDCapTidieR/articles/glossary.html#repeating) all instruments together or [repeats](https://chop-cgtinformatics.github.io/REDCapTidieR/articles/glossary.html#repeating) instruments independently of each other. From REDCap, this looks like:
```

Comment:
We may have to update the repeat and non-repeat links if we end up going with the version of the glossary that has separate "repeating-event" and "repeating-instrument" entries but that all may change

Reply:
Sure thing, I just wanted to start getting the hyperlinks implemented since there's so many.